lib/App/DuckPAN/Query.pm

```diff
@@ -2,95 +2,125 @@ package App::DuckPAN::Query;
 # ABSTRACT: Main application/loop for duckpan query
 
 use Moo;
-
-my $query;
-my $history_path;
-
+use DDG;
+use DDG::Request;
```

Comment:
Sorry, that's wrong. I should have just gone to sleep last night ;)

The way DuckPAN ensures DDG is installed, is that it checks for it to be installed thanks to:
(check_requirements)[https://github.com/duckduckgo/p5-app-duckpan/blob/076547d8a020afcbbee880d76d16411a456d082e/lib/App/DuckPAN.pm#L375] which calls (check_ddg)[https://github.com/duckduckgo/p5-app-duckpan/blob/076547d8a020afcbbee880d76d16411a456d082e/lib/App/DuckPAN.pm#L490] and then prompts the user to run `duckpan DDG` if DDG is not installed.